BP/localization/english/religion/BP_religion_l_english.yml

```diff
@@ -0,0 +1,6 @@
+﻿l_english:
+ # WTSMS Mazdan Family
+ rf_mazdan:0 "Mazdan"
+ #Hostility Doctrines
+ mazdan_hostility_doctrine_name:0 "Mazdan"
+ mazdan_hostility_doctrine_desc:0 "The following religions are Mazdan:\n[GetReligionFamily('rf_mazdan').GetReligions]"
```

Comment:
Could you rename this file into BP_religion_mazdan_l_english? Otherwise the name might be a bit confusing, I think

Reply:
I can rename it; I took the name because specifically to mirror the base game, in which religion_l_english.yml contains the localization for hostility doctrines. I prefer to match base game naming schema where possible, but I'll change it if you prefer.

Reply:
Ok, that makes sense, I didn't realise you were mirroring the base game